### ISSUE TITLE
Training result is abnormal without pre-trained weights.

### ISSUE THREAD

**Issue by KevinRyu** [jmpap/YOLOV2-Tensorflow-2.0#6]
Hi,

Thanks for sharing your Yolo_v2 implementation using tf 2.0, first. 
I trained this neural network after remarking "Load Pre-trained weights" code lines.
But both main and validation loss is not decreased at all. I got a loss graph looks like spike.
I think training is not proceeded correctly. Would you please give me an advice to solve this problem?
For example, I need to adjust hyper parameters, and so on.

**Comment by jmpap** [comment]
Hi,
I tried to train the model without pre-trained weights. Train loss decreases but validation loss increases. I am not sure it'is possible to train yolo v2 on our own dataset without pre-trained weights. According to the original article, the author explains that the pre-trained weights come from a classification model trained on ImageNet.

**Comment by KevinRyu** [comment]
Hi,

Thanks for your comment. I misunderstood code lines for loading pretrained weight (output for inference) that you have already been training with your data. I got it. I'll restore remarked code lines again and train it with custom data. By the way, you said train loss decreases. But I didn't see it. In my case, both train and validation loss is abnormal. I didn't understand your codes correctly. I will try again.

**Comment by jmpap** [comment]
Hi, if your train loss increases or is unstable, you can try a lower learning rate.

**Comment by KevinRyu** [comment]
OK! I will try. I learned many important points what I didn't know from
your code. Your code is very clean and nice. Thank you very much.

2020년 4월 25일 (토) 오전 4:20, jmpap <notifications@github.com>님이 작성:

**Comment by jmpap** [comment]
Thank your for your appreciation!
